src/js/script.js

```diff
@@ -116,17 +116,31 @@
       });
 
       item.querySelectorAll('[data-step]').forEach(arrow => {
-        const control = arrow.dataset.controls;
-        const step    = arrow.dataset.step;
-        const input   = document.getElementById(control);
+        const step = arrow.dataset.step;
 
         arrow.addEventListener('click', () => {
+          const isPortrait = matchMedia('screen and (orientation: portrait)').matches;
+          let control      = arrow.dataset.controls;
+
+          if (isPortrait) {
+            if (control.slice(-1) === 'x') {
+              control = control.slice(0, -1) + 'y';
+            } else {
+              control = control.slice(0, -1) + 'x';
+            }
+          }

```

Comment:
This part might be slightly shorter I think:

```
if (isPortrait) {
    const axis = (control.slice(-1) === 'x') ? 'y' : 'x';
    control = control.slice(0, -1) + axis;
}
```
 But it requires to use a ternary operator, so it’s up to you — but I can see you're using it above :)

I'll test this and could merge if that works ;)

Reply:
Looks much better and, I just tested it and, it still works. Thank you.